src/components/Navbar.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const Navbar = () => {
```

Comment:
Could you change to `NavBar`, please?
```suggestion
const NavBar = () => {
```